processing/src/test/java/io/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -1976,7 +1976,10 @@ public void testGroupByWithAlphaNumericDimensionOrder()
         .setDimensions(
             Lists.<DimensionSpec>newArrayList(
                 new ExtractionDimensionSpec(
-                    "quality", "alias", new LookupExtractionFn(new MapLookupExtractor(map, false), false, null, false, false), null
+                    "quality",
```

Comment:
unless there are any code changes, I would rather keep reformatting for unrelated files out of this PR.